canonical/ubuntu.com
# Amend .inline-logos to bring inline with vf

# Done

Amend .inline-logos to bring inline with vf
- delete old tools folder
- add cloud redirects
## QA

Requires cutting edge vanilla-framework and ubuntu-vanilla-theme
Run make run then make sure the inline-logos look okay.
## Details

Card: https://trello.com/c/HqCSLVE1
